react/features/app/components/Root.web.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+
+import { App } from './App';
+
+/**
+ * React component for root component.
+ *
+ * @class Root
+ * @extends Component
+ */
+export default class Root extends Component {
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     */
+    render() {
+
+        // Explicitly pass window location.
+        return (
+            <App url = { window.location.toString() } />
```

Comment:
The `url` from `window.location` does not have to be passed to the `Web` and I don't want third-party developers to see that in our source code and think that they have to do it.